configure.ac

```diff
@@ -397,7 +397,7 @@ AC_ARG_ENABLE(request-early,
 
 # Enable duplicate rules id
 AC_ARG_ENABLE(unique_id,
-              AS_HELP_STRING([--enable-unique_id],
+              AS_HELP_STRING([--enable-not_unique_id],
```

Comment:
yes please. You may want to perform a "git commit --amend" to avoid unnecessary commits in the repository.